.gitignore

```diff
@@ -19,4 +19,4 @@ pkg
 /railties/test/initializer/root/log
 /railties/doc
 /railties/tmp
-/guides/output
+/guides/output
```

Comment:
This is diff noise, it would be nice if this was reverted to minimize the noise.